### DISCUSSION TITLE
USB power control on Hero9 whilst mediamod connected

### DISCUSSION THREAD

**Discussion by andyb2000** [gopro/labs#1208] [Feature Requests]
Hi all,
As many others encounter, I have problems with the mediamod and power events. Running labs fw v1.72.70 I've used the USB power control to try and disable all usb power events as I don't want any usb power, plug/unplug events from powering on the camera. However none of these seem to work.
Is this because the usb is being plugged into the mediamod and not camera direct? Would there be a way in labs to detect this and still not allow any usb power events?

(A little more behind the reason. If powered via usb whilst doing a recording, often the power on by USB will cause the mediamod not to record audio. So frustrating! So I lose a whole video of recording due to no audio!! I know this is a bug in mediamod that seemingly is very common, but I wondered if labs can help out here!)

Thanks.

**Comment by dnewman-gpsw** [comment]
I can't think of anything within Labs that might help.

**Comment by andyb2000** [comment]
Thank you, I thought that might be the case but appreciate the reply.
